### PR TITLE
Fix goreport warnings of ineffassign and misspell

### PR DESCRIPTION
**Pull request by ashokponkumar**
Signed-off-by: Ashok Pon Kumar <ashokponkumar@gmail.com>

### CODE REVIEW

File: libcontainer/cgroups/fs/cpuacct.go [hunk 1]
```diff
@@ -83,8 +83,8 @@ func (s *CpuacctGroup) GetStats(path string, stats *cgroups.Stats) error {
 
 // Returns user and kernel usage breakdown in nanoseconds.
 func getCpuUsageBreakdown(path string) (uint64, uint64, error) {
-	userModeUsage := uint64(0)
-	kernelModeUsage := uint64(0)
+	var userModeUsage uint64
+	var kernelModeUsage uint64
```
Comment (kolyshkin) [inline libcontainer/cgroups/fs/cpuacct.go:87]:
```suggestion
	var userModeUsage, kernelModeUsage uint64
```

**Review by kolyshkin — DISMISSED** [review]
one nit, otherwise LGTM

File: libcontainer/cgroups/fs/cpuacct.go (hunk 1 above)
Comment (ashokponkumar) [reply libcontainer/cgroups/fs/cpuacct.go]: Thanks @kolyshkin . Have made the changes.

**Review by AkihiroSuda — APPROVED** [review]

**Review by cyphar — APPROVED** [review]
LGTM.

**Pull request closed without merging**